src/controllers/events/delete_event.js

```diff
@@ -0,0 +1,26 @@
+const Request = require('request');
```

Comment:
This file is virtually identical to `delete_place` - do you think we should make a generic file for deleting data, or keep them separate @m4v15 ?

Reply:
Yeah and maybe extract the place/event/product from the url?

Reply:
I mean theoretically a lot of these pages will be pretty similar, could make the get_all page generic at some point, but that would require a bit more of a rethink @JWLD

Reply:
I think we'll leave it separate for now and refactor later